src/controllers/authController.js

```diff
@@ -110,8 +111,8 @@ module.exports.token = async (req, res) => {
       name: userResalt.rows[0].name,
       email: userResalt.rows[0].email,
     },
-    process.env.JWT_SECRET,
-    { expiresIn: "1h" }
+    privateKey,
```

Comment:
The way it works now, it generates a new pair of keys every time an authentication happens. We shouldn't do this. We should only generate a key pair at the start up of the application, and should store it in the `signing_keys` table. And we should only generate the key pair only if there is no active key pair is in the table. 

Then we can retrieve the private and public keys at the point of signing the token and validating the token.